### DISCUSSION TITLE
Is there a way to add certificate?

### DISCUSSION THREAD

**Discussion by shasi24** [tiangolo/uwsgi-nginx-docker#222] [Questions] [answered]
Hi, 
I want to add certificate to the wsgi application
not sure if its suppose to be added at wsgi.conf or the nginx.conf
any help would be appreciated

**Comment by fegmorte** [comment]
Hi,

Depending on where you are going to deploy your application.
Either in nginx configuration or in your cloud provider.

**Comment by bmm-alc** [comment, marked as answer]
certificates are like configuration, hence should not be included in the container, but rather handled by a load-balancer or an ingress server.
